movieApp.js

```diff
@@ -0,0 +1,181 @@
+import { movies } from "./movieArr.js";
+
+const nowStreaming = document.querySelector(".now-streaming");
+
+function createMovieCard(movie) {
+  //Creat a cotainer to store the Image and readMore Button
+
+  const movieContainer = document.createElement("div");
+  movieContainer.className = "movie-container";
+
+  //
+  //  Create an img element and set its src attribute
+  const img = document.createElement("img");
+  img.width = 200;
+  img.src = movie.poster_url;
+
+  // Create a button element
+  const readMoreBtn = document.createElement("button");
+  readMoreBtn.textContent = "Read More";
+  readMoreBtn.className = "accordion";
+
+  // starRating
+  const starRating = document.createElement("div");
+  starRating.className = "star-rating";
+  const numberOfStars = 5;
+  let stars = [];
+  for (let x = 0; x < numberOfStars; x++) {
+    const star = document.createElement("i");
+    star.className = "rating__star far fa-star";
+    starRating.appendChild(star);
+    stars.push(star);
+  }
+  const starClassActive = "rating__star fas fa-star";
+  const starClassInactive = "rating__star far fa-star";
+  stars.forEach((star, clickedStarIndex) => {
```

Comment:
nice one here, especially with the double loop and use of index!

Reply:
Thanks